src/Entity/City.php

```diff
@@ -41,6 +40,10 @@ public function __toString()
      */
     private $city;
 
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $city_address;
```

Comment:
I think there shoud be getters and setters

Reply:
During our meeting, we decided to move address column to programEvents entity.